gateway/assets/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <title>OpenFaaS Portal</title>
     <link href="https://fonts.googleapis.com/css?family=Rationale|Roboto+Mono|Roboto:300,400,400i,500,700" rel="stylesheet">
```

Comment:
@kenfdev can we do anything for this? What would happen in a fully offline mode?